doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -1055,7 +1101,7 @@ Wi-Fi
 
 The issues in this section are related to the :ref:`ug_wifi` protocol.
 
-.. rst-class:: v2-7-0
+.. rst-class:: v2-8-0 v2-7-0
 
 SHEL-2878: WPA3 security mode not working
   WPA3 security mode is not working for nRF54L15 PDK and nRF7002 EB.
```

Comment:
- NCSDK-29650 : Wi-Fi connection times are increased in `_ns` builds. Increased the connection timeout (10s -> 15s) to work around this.
- NCSDK-29649 : Flash firmware integrity check doesn't work due to MbedTLS dependencies. Due to this nRF70 patches cannot be validated and corrupted patches can manifest in nRF70 bootup failures
- NCSDK-29651 : 54H20 Legacy crypto not supported. PSA crypto is used by default which limits Wi-Fi security only up to WPA2.
- SHEL-3226 : In nRF70 driver page docs, link to doxygen API has been removed